themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties

```diff
@@ -0,0 +1,224 @@
+doLogIn=Inloggen
+doRegister=Registeer
+doCancel=Annuleer
+doSubmit=Verzenden
+doYes=Ja
+doNo=Nee
+doContinue=Doorgaan
+doAccept=Accepteren
+doDecline=Afwijzen
+doForgotPassword=Wachtwoord vergeten?
+doClickHere=Klik hier
+doImpersonate=Belichamen
+kerberosNotConfigured=Kerberos niet geconfigureerd
+kerberosNotConfiguredTitle=Kerberos niet geconfigureerd
+bypassKerberosDetail=U bent niet ingelogd via Kerberos of uw browser kan geen Kerberos login aan. Klik op 'doorgaan' om via een andere manier in te loggen
+kerberosNotSetUp=Kerberos is niet goed geconfigureerd. U kunt niet inloggen.
+registerWithTitle=Registeer met {0}
+registerWithTitleHtml={0}
+loginTitle=Log in naar {0}
+loginTitleHtml={0}
+impersonateTitle={0} Belichaam gebruiker
+impersonateTitleHtml=<strong>{0}</strong> Belichaam gebruiker</strong>
+realmChoice=Realm
+unknownUser=Onbekende gebruiker
+loginTotpTitle=Mobile Authenticator Setup
```

Comment:
yes, but then I'd have to translate Authenticator as well (at least, that's how it feels to me). "Mobile Authenticator" or "Mobiele Authenticatie-app" (or sth like that). That's why I kept it close to the original and close to what I think most users might know.